gravity_upgrade.md

```diff
@@ -72,25 +73,154 @@ A manual upgrade operation starts with an operation plan which is a tree of acti
+feature showing the status changes over time which may be helpful to identify
+root causes or brief fluctuations of the cluster state:
+
+```
+ubuntu@telekube0:~/i$ sudo gravity status history
+2020-07-30T16:18:08Z [Leader Elected]   new leader 172.28.128.3
+2020-07-30T16:18:35Z [Node Degraded]    node=172_28_128_3.pensivevillani4966
+2020-07-30T16:19:13Z [Probe Failed]     node=172_28_128_3.pensivevillani4966    checker=dns
+2020-07-30T16:19:13Z [Probe Succeeded]  node=172_28_128_3.pensivevillani4966    checker=node-status
+2020-07-30T16:19:13Z [Probe Failed]     node=172_28_128_3.pensivevillani4966    checker=etcd-healthz
+2020-07-30T16:19:35Z [Probe Succeeded]  node=172_28_128_3.pensivevillani4966    checker=etcd-healthz
+2020-07-30T16:19:35Z [Probe Succeeded]  node=172_28_128_3.pensivevillani4966    checker=dns
+2020-07-30T16:19:35Z [Node Healthy]     node=172_28_128_3.pensivevillani4966
+2020-07-30T16:29:38Z [Probe Failed]     node=172_28_128_3.pensivevillani4966    checker=etcd-healthz
+2020-07-30T16:29:38Z [Node Degraded]    node=172_28_128_3.pensivevillani4966
+2020-07-30T16:30:06Z [Probe Succeeded]  node=172_28_128_3.pensivevillani4966    checker=etcd-healthz
+2020-07-30T16:30:06Z [Node Healthy]     node=172_28_128_3.pensivevillani4966
+2020-07-30T16:30:57Z [Node Degraded]    node=172_28_128_3.pensivevillani4966
+2020-07-30T16:31:07Z [Probe Failed]     node=172_28_128_3.pensivevillani4966    checker=etcd-healthz
+2020-07-30T16:32:05Z [Probe Succeeded]  node=172_28_128_3.pensivevillani4966    checker=etcd-healthz
+2020-07-30T16:32:05Z [Node Healthy]     node=172_28_128_3.pensivevillani4966
+2020-07-30T16:32:46Z [Probe Failed]     node=172_28_128_3.pensivevillani4966    checker=node-status
+2020-07-30T16:32:46Z [Probe Failed]     node=172_28_128_3.pensivevillani4966    checker=etcd-healthz
+2020-07-30T16:33:07Z [Probe Succeeded]  node=172_28_128_3.pensivevillani4966    checker=node-status
+2020-07-30T16:34:05Z [Probe Succeeded]  node=172_28_128_3.pensivevillani4966    checker=etcd-healthz
+2020-07-30T16:34:05Z [Node Healthy]     node=172_28_128_3.pensivevillani4966
+```
+
+It's also worth mentioning that from version 5.5.50 on, a few minor improvements
+were also added to the `gravity status` output. To mention a few:
 
+* `gravity status` now displays unhealthy critical system pods and some
+  visibility into the controller status loop if the cluster is in a degraded
+  state
+* a new set of pre-upgrade checks were added to verify if previous upgrade
+  operation succeeded or a failed plan was rolled back
 
-## Exploring a Gravity Manual Upgrade
+* another set of pre-upgrade checks was added to verify if any Teleport node is
+  unavailable
 
+* `gravity plan resume` now launches its execution in background by default, but a new `--block` option was added to resume in foreground and a different command `gravity plan watch` was also introduced to track the plan progress
```

Comment:
The command to follow the plan progress is actually `gravity plan --tail`, not watch.